doc/man1/openssl-enc.pod.in

```diff
@@ -417,6 +429,12 @@ Base64 decode a file then decrypt it using a password supplied in a file:
  openssl enc -aes-256-ctr -pbkdf2 -d -a -in file.aes256 -out file.txt \
     -pass file:<passfile>
 
+AES key wrapping:
+
+ openssl enc -e -a -id-aes128-wrap-pad -K 000102030405060708090A0B0C0D0E0F -in file.bin
+ or 
```

Comment:
Should the _or_ not have a space at the front?

Reply:
I will add a space.

Reply:
I meant drop the leading spaces so it's not a mono-space font.